### PR TITLE
CUMULUS-2542

### PR DESCRIPTION
**Pull request by Nnaga1**
**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

Added a new file for the ASYNC dropdown in the Dropdowns directory, which is to be used in the Collections page. We need this new dropdown because the current one (SimpleDropdown), does not show all of the collection options possible. I also added about 70 collections in the seed for testing purposes.

* Detailed list or prose of changes
* ...

## PR Checklist

- [x] Update CHANGELOG
- [x] Unit tests
- [x] Adhoc testing
- [x] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -28,8 +28,16 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
...
+<<<<<<< HEAD
+  - Created a new ASYNC dropdown in order to be used to search Collections since some Collections were not showing
+  - Created a new file for styles to be imported in Simple-Dropdown and Async-Dropdown
 - **CUMULUS-2534**
-  - Added a copy button to errors pages so the error can be copied and searched up on the internet. 
+  - Added a copy button to errors pages so the error can be copied and searched up on the internet.
+=======
+  - Created an ASYNC dropdown component to be used in the page when a Collection is selected since it was initially not showing all options
+  - Created a new style utils file which is imported into and used in Async-Dropdown and Simple-Dropdown
+>>>>>>> ece0893a1e66875ba2f755a7a3bc6ef474628ea2
```
Comment (dopeters) [inline CHANGELOG.md:40]: Can we clean this up? Looks like some merge conflict markup is still there

File: app/src/js/components/DropDown/async-dropdown.js [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+          <label htmlFor={id}>{label}</label>
+        </li>
+        <li className="dropdown__element">
+          <Async
+            aria-label={label}
+            blurInputOnSelect={true}
+            {...rest}
+            className={className}
+            components={ { DropdownIndicator } }
+            style={customStyles}
```
Comment (dopeters) [inline app/src/js/components/DropDown/async-dropdown.js:54]: this should be `styles={customStyles}`

File: cypress/fixtures/seeds/collectionsFixture.json [hunk 3]
```diff
@@ -193,4 +193,4 @@
     }
     
   ]
-}
+}
```
Comment (dopeters) [inline cypress/fixtures/seeds/collectionsFixture.json:196]: Can we revert this please? (Just add back the new line)

File: CHANGELOG.md [hunk 4]
```diff
@@ -28,8 +28,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
     logs, likewise use a pattern like the ES\_DISTRIBUTION\_TARGET\_PATTERN.
 
 ### Changed
+- **CUMULUS-2542**
+  - Created a new ASYNC dropdown in order to be used to search Collections since some Collections were not showing
+  - Created a new file for styles to be imported in Simple-Dropdown and Async-Dropdown
```
Comment (dopeters) [inline CHANGELOG.md:33]: Can you move this so that the ticket numbers are in numerical order?

**Review by dopeters — CHANGES_REQUESTED** [review]
Great work! Just some very tiny tweaks needed, but the functionality works great.

File: CHANGELOG.md (hunk 4 above)
Comment (Nnaga1) [reply CHANGELOG.md]: Fixed

File: cypress/fixtures/seeds/collectionsFixture.json (hunk 3 above)
Comment (Nnaga1) [reply cypress/fixtures/seeds/collectionsFixture.json:196]: fixed

File: app/src/js/components/DropDown/async-dropdown.js (hunk 2 above)
Comment (Nnaga1) [reply app/src/js/components/DropDown/async-dropdown.js:54]: fixed

File: app/src/js/utils/dropdown-utils.js [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+const customStyles = {
+  control: (base) => ({
+    ...base,
+    border,
+    borderRadius,
+    boxShadow,
+  }),
+  menu: (base) => ({
+    ...base,
+    width: 100,
```
Comment (dopeters) [inline app/src/js/utils/dropdown-utils.js:28]: Not something you did, but can we get rid of this width? Causing some issues with the async dropdown menu

**Review by dopeters — APPROVED** [review]
Thanks for all the changes! Looks great

**Pull request closed without merging**